contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+1 row in set (0.00 sec)
+```
+
+Also verify that chainquery is again happy. View the chainquery logs:
+
+```
+k8s-lbry logs chainquery
+```
+
+Press Ctrl-C to quit viewing the logs.
+
+### lbrynet API service (not for spee.ch)
+
+This is for a standalone lbrynet API service inside your cluster. Blob storage
+goes to its own persistent volume, but is configured with `save_files=false`.
+There is no outside access to the Downloads directory provided. You can stream
+blobs from lbrynet via `http://lbrynet:5279/get/CLAIM_NAME/CLAIM_ID`.
+
+This particular lbrynet configuration won't work for spee.ch (v0.5.12). spee.ch
+needs direct access to the Downloads directory of lbrynet. **If you are wanting
+lbrynet for spee.ch, skip this section, and head directly to the [spee.ch
+section](#speech-and-lbrynet-sidecar-and-mysql), which implements its own
+lbrynet sidecar.**
+
+Edit `values-dev.yaml` and set `lbrynet.enabled` to `true`.
+
+Upgrade the release to turn on lbrynet:
+
+```
+k8s-lbry upgrade
+```
+
+You can view the lbrynet logs:
+
+```
+k8s-lbry logs lbrynet
+```
+
+#### IMPORTANT - Backup your cluster wallet
+
+The wallet is created inside the `lbrynet` persistent volume.
+
+Copy the wallet in case the volume gets destroyed:
+
+```
+k8s-lbry lbrynet-copy-wallet /tmp/k8s-lbry-lbrynet-wallet-backup.json
+```
+
+Check the contents of `/tmp/k8s-lbry-lbrynet-wallet-backup.json` and move the
+file to a safe place for backup (make sure to delete the temporary file.)
+
+Once your wallet is backed up, you can generate a receiving address in order to
+deposit LBC:
+
+```
+k8s-lbry lbrynet address unused
+```
+
+### spee.ch (and lbrynet sidecar and mysql)
+
+*Note: Throughout this deployment, the unstylized name `speech` is used.*
+
+Speech needs three containers, running in two pods:
+
+ * `speech` pod:
+ 
+   * speech, the nodejs server container.
+
+   * lbrynet, running in the same pod as speech, so as to share one downloads
+     directory. (This is called a 'sidecar' container, which is guaranteed to
+     run on the same kubernetes node as the spee.ch container.)
+   
+ * `speech-mysql` pod:
+ 
+   * mysql for storing the speech database.
+
+Edit `values-dev.yaml`. 
+
+ * Set `speech-mysql.enabled` to `true`.
+ * Set `speech.enabled` to `true`.
+ * Set `speech.service.hostname` to your subdomain name for speech.
+ * Set `speech.site.details.host` to your subdomain name for speech.
+ * Set `speech.site.details.ipAddress` to your Load Balancer external IP address.
+ * Set `speech.site.details.title`
+
+Upgrade the release to turn on `speech`, `speech-lbrynet`, and `speech-mysql`:
+
+```
+k8s-lbry upgrade
+```
+
+Speech will not work yet! Continue on through the next sections.
+
+#### IMPORTANT - Backup your speech wallet
+
+The wallet for speech is created inside the `speech-lbrynet` persistent volume.
+
+Copy the wallet in case the volume gets destroyed:
+
+```
+k8s-lbry speech-lbrynet-copy-wallet /tmp/k8s-lbry-speech-lbrynet-wallet-backup.json
```

Comment:
Could we potentially `cat` and pipe this into `apt-get install qrencode` on the host somehow so the user could scan it into a wallet app or something for backup?

Reply:
I don't think I want to introduce a new dependency, but adding instructions to do this is a great idea!

Reply:
actually if the dependency is only for one subcommand, maybe its ok. Lemme think about this. Its a grand idea.

Reply:
OK after thinking about it, I think this is the wrong place to implement this. I feel like lbrynet itself should have this feature. Otherwise someone might look at the lbrynet docs, and not know what to do with this qrcode.